src/pages/LogInWithShortLivedAuthTokenPage.js

```diff
@@ -85,7 +94,15 @@ function LogInWithShortLivedAuthTokenPage(props) {
                 <Text style={[styles.textHeadline, styles.textXXLarge]}>{props.translate('deeplinkWrapper.launching')}</Text>
                 <View style={styles.mt2}>
                     <Text style={[styles.fontSizeNormal, styles.textAlignCenter]}>
-                        {props.translate('deeplinkWrapper.expired')} <TextLink onPress={() => Navigation.navigate()}>{props.translate('deeplinkWrapper.signIn')}</TextLink>
+                        {props.translate('deeplinkWrapper.expired')}{' '}
```

Comment:
NAB, we can move {' '} to new line

Reply:
This change was from running `npm run prettier`